src/lib389/setup.py

```diff
@@ -30,6 +31,18 @@
 with open(path.join(here, 'README.md'), 'r') as f:
     long_description = f.read()
 
+#
+# For some historical reason when using prefix install
+#  files that should be in /usr/sbin/, /usr/share/ are
+#  in $PREFIX/sbin, $PREFIX/share
+# So lets mimick this behavior
+
+prefix=environ.get('INSTALL_PREFIX', '/usr')
+print(f"argv={argv}")
```

Comment:
Is it a debug statement?

Reply:
Indeed it is displayed in the output of `make lib389-install` that said, it is also present in the main branch